tests/Catch2_Tests/comparators/approval_tests/ComparatorDisposerTest.ComparatorDisposer_object_can_be_moved_without_disposing_a_comparator_twice.approved.txt

```diff
@@ -0,0 +1 @@
+bar
```

Comment:
If this test continues to use an output file, could its file name be shortened by at least this many characters please: `comparator_twice.approved.txt`

**Background**: I've previously had problems cloning this reopen Windows, because of long file names of approval files exceeding the path length limit - and so having to rename the cloned folder to shorten the filename.

Because of this, I recently shortened the names of the longest files here.

Before this commit, `git ls-files` tells me the longest filename was:

`tests/DocTest_Tests/docs/approval_tests/DocTestDocsSamples.ApprovalTests-MultipleOutputFiles-AutoApproving.American.approved.txt`

This name is a fair bit longer:

`tests/Catch2_Tests/comparators/approval_tests/ComparatorDisposerTest.ComparatorDisposer_object_can_be_moved_without_disposing_a_comparator_twice.approved.txt`

Reply:
Now the "approved" file is written at runtime.